pkg/artifactory/datasource/repository/local/datasource_artifactory_local_repository_test.go

```diff
@@ -63,3 +65,682 @@ func mkTestCase(repoType string, t *testing.T) (*testing.T, resource.TestCase) {
+		csFt9MUgxz/xov2qaj7aC+bc3k7msUVaRLardpAkZ09AUrQyQGRWf50/XPUu+dO4
+		5iYxVu6OH/uIa664k6qDwAECgYEAslS8oomgEL3VhbWkx1dLA5MMggTPfgpFNsMY
+		4Y4JXcLrUEUgjzjEvW0YUdMiLhP8qapDSiXxj1D3f9myxWSp8g0xc9UMZEjCZ9at
+		RcjNyP8zBLnCKqokSt6B3puyDsnvvrC/ugIBbnTFBOCJSZG7J7CwJx8z3KbQI1ub
+		+fpCj7ECgYAd69soLEybUGMjsdI+OijIGoUTUoZGXJm+0VpBt4QJCe7AMnYPfYzA
+		JnEmN4D7HLTKUBklQnb/FhP/RuiT2bSAd1l+PNeuU7gYROCBBonzxXQ1wEbNrSYA
+		iyoc9g/kvV8HTW8361xEhu7wmuSEEx1gQ/7sdhTkgrTncc8hxVRxuA==
+		-----END RSA PRIVATE KEY-----
+		EOF
+			public_key = <<EOF
+		-----BEGIN PUBLIC KEY-----
+		MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA2ymVc24BoaZb0ElXoI3X
+		4zUKJGZEetR6F4yT1tJhkPDg7UTmiNoFB5TZJvP6LBrrSwszkpZbxaVOkBrwrGbq
+		FUaXPgud8VabfHl0imXvN746zmpjYEMGqJzm+Gh6aBWOlnPdLuHhds/kcanFAEpp
+		j5yN0tVWDnqjOJjR7EpxMSdP3TSd6tNAY73LGNLNJQc6tSxh8nMIb4HhSWQSgfof
+		+FwcLGvs+mmyBq8Jz5Zy4BSCk1fQFmCnSGyzpyBD0vMd6eLHk2l0tm56DrlonbDM
+		X8KGs7e9ZgjANkT5PnipLOaeLJU4H+OWyBZUAT4hl/iRVvLwV81x7g0/O38kmPYJ
+		DQIDAQAB
+		-----END PUBLIC KEY-----
+		EOF
+			lifecycle {
+				ignore_changes = [
+					private_key,
+					passphrase,
+				]
+			}
+		}
+		resource "artifactory_local_alpine_repository" "{{ .repo_name }}" {
+			key 	     = "{{ .repo_name }}"
+			primary_keypair_ref = artifactory_keypair.{{ .kp_name }}.pair_name
+			depends_on = [artifactory_keypair.{{ .kp_name }}]
+		}
+
+		data "artifactory_local_alpine_repository" "{{ .repo_name }}" {
+			key = artifactory_local_alpine_repository.{{ .repo_name }}.id
+		}
+	`, map[string]interface{}{
+		"kp_id":     kpId,
+		"kp_name":   kpName,
+		"repo_name": name,
+	}) // we use randomness so that, in the case of failure and dangle, the next test can run without collision
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { acctest.PreCheck(t) },
+		ProviderFactories: acctest.ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: localRepositoryBasic,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(fqrn, "key", name),
+					resource.TestCheckResourceAttr(fqrn, "package_type", "alpine"),
+					resource.TestCheckResourceAttr(fqrn, "primary_keypair_ref", kpName),
+					resource.TestCheckResourceAttr(fqrn, "repo_layout_ref", func() string { r, _ := repository.GetDefaultRepoLayoutRef("local", "alpine")(); return r.(string) }()), //Check to ensure repository layout is set as per default even when it is not passed.
+				),
+			},
+		},
+	})
+}
+
+func TestAccDataSourceLocalCargoRepository(t *testing.T) {
+	_, fqrn, name := test.MkNames("cargo-local", "data.artifactory_local_cargo_repository")
+	params := map[string]interface{}{
+		"anonymous_access": test.RandBool(),
+		"name":             name,
+	}
+	localRepositoryBasic := util.ExecuteTemplate("TestAccLocalCargoRepository", `
+		resource "artifactory_local_cargo_repository" "{{ .name }}" {
+		  key                 = "{{ .name }}"
+		  anonymous_access = {{ .anonymous_access }}
+		}
+
+    data "artifactory_local_cargo_repository" "{{ .name }}" {
```

Comment:
Let's have consistent indentation *within* the template 😄 (this happens a number of times in this file).

Reply:
Damn thought I made consistent the intellij indentation settings, guess I didn't fix it. Will check again